src/components/ids-checkbox-group/ids-checkbox-group.js

```diff
@@ -0,0 +1,63 @@
+import { customElement, scss } from '../../core/ids-decorators';
+import { attributes } from '../../core/ids-attributes';
+import Base from './ids-checkbox-group-base';
+
+import styles from './ids-checkbox-group.scss';
+
+/**
+ * IDS Checkbox Group Component
+ * @type {IdsCheckboxGroup}
+ * @inherits IdsElement
+ * @mixes IdsEventsMixin
+ * @mixes IdsThemeMixin
+ * @part checkbox-group - the checkbox-group element
+ */
+@customElement('ids-checkbox-group')
+@scss(styles)
+export default class IdsCheckboxGroup extends Base {
+  constructor() {
+    super();
+  }
+
+  /**
+   * Return the attributes we handle as getters/setters
+   * @returns {Array} The attributes in an array
+   */
+  static get attributes() {
+    return [
+      attributes.LABEL,
+    ];
+  }
+
+  /**
+   * Create the Template for the contents
+   * @returns {string} The Template
+   */
+  template() {
+    return `
+      <div class="ids-checkbox-group" part="checkbox-group">
+        <ids-text font-size="16" type="h1">${this.label}</ids-text>
```

Comment:
```suggestion
        <ids-text font-size="16" type="span">${this.label}</ids-text>
```

Dont think we need this to be an h1.. Try with span?